modules/installation-aws-upi-requirements.adoc

```diff
@@ -0,0 +1,399 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-requirements-{context}"]
+= Required AWS UPI infrastructure components
+
+To install {product-title} on user-provisioned infrastructure in
+Amazon Web Services (AWS), you must manually create both the machines and their
+supporting infrastructure.
+
+You can use the provided CloudFormation templates to create this infrastructure,
+or you can manually create the components. Review the CloudFormation templates
+for more details about how the components interrelate.
+
+[id="installation-aws-upi-cluster-machines-{context}"]
+== Cluster machines
+
+Create `AWS::EC2::Instance` objects for the following machines:
+
+* A bootstrap node. This machine is required during installation, but you
+can remove it after your cluster deploys.
+* At least three control plane machines. The control plane machines are not
+governed by a machineset.
+* Worker nodes. You can create these manually during the cluster installation
+process or you can create and control them by using a MachineSet after
+you create a bootstrap node.
+
+[id="installation-aws-upi-other-infrastructure-{context}"]
+== Other infrastructure components
+
+* A VPC
+* DNS entries
+* Load balancers and listeners
+* A Route53 zone
+* Security groups
+* IAM roles
+* S3 buckets
+
+.Required VPC components
+
+You need to create a VPC and subnets that allow communication to your machines.
+
+[cols="2a,7a,3a,3a",options="header"]
+|===
+
+|Component
+|AWS type
+2+|Description
+
+|VPC
+|* `AWS::EC2::VPC`
+* `AWS::EC2::VPCEndpoint`
+2+|You must create a public VPC for the cluster to use. The VPC requires an
+endpoint that references the route tables for each subnet.
+
+|Public subnets
+|* `AWS::EC2::Subnet`
+* `AWS::EC2::SubnetNetworkAclAssociation`
+2+|Your VPC must create public subnets for between 1 and 3 availability zones
+and associate them with appropriate ingress rules.
+
+|Internet gateway
+|
+* `AWS::EC2::InternetGateway`
+* `AWS::EC2::VPCGatewayAttachment`
+* `AWS::EC2::RouteTable`
+* `AWS::EC2::Route`
+* `PublicSubnetRouteTableAssociation`
+* `AWS::EC2::NatGateway`
+* `AWS::EC2::EIP`
+2+|You must have a public internet gateway, with public routes, attached to the
+VPC. Each public subnet must also be attached to the route and have a NAT
+gateway and EIP address.
+
+.7+|Network access control
+.7+| * `AWS::EC2::NetworkAcl`
+* `AWS::EC2::NetworkAclEntry`
+2+|You must allow the VPC to access the following ports:
+h|Port
+h|Reason
+
+|`80`
+|Inbound HTTP traffic
+
+|`443`
+|Inbound HTTPS traffic
+
+|`22`
+|Inbound SSH traffic
+
+|`1024` - `65535`
+|Inbound ephemeral traffic
```

Comment:
This is what the current installer CloudFormation templates have, but I don't know that such a wide range is actually required.  I'll see about getting better installer-side docs for these rules, and ping you once we have something.

Reply:
I'd forgotten to follow up here, but openshift/installer#2296 removed these blocks from the installer's CloudFormation stacks.